### PR TITLE
Version 0.3 sample

### PR DESCRIPTION
**Pull request by will-moore**
Added a sample v0.3 file, generated by copying the v0.2 file and making some changes since `bioformats2raw` doesn't generate v0.3 data yet. Can be updated when it does.

Also added info to README and added to the checksum test check.
Do we want to check different values based on version number here?

The v0.3 data passes all the existing tests (info, download and "checksum") since they don't assume data is 5D. Also, the v0.3 data passes even without the `axes` metadata required by the spec. I guess this is OK since the job of `ome-zarr-py` is not to validate, but do we want to add this somewhere (either in `ome-zarr-py` or in this test repo)?

### CODE REVIEW

**Comment by joshmoore** [comment]
> I guess this is OK since the job of `ome-zarr-py` is not to validate, but do we want to add this somewhere (either in `ome-zarr-py` or in this test repo)?

Rather than add logic to this repo, I would error on the side of an `ome_zarr validate` method (if not just a part of `info`)

File: sources.yml [hunk 1]
```diff
@@ -12,9 +12,6 @@
 
 # Generally organized by speed
 
-- name: create
-  script: ome_zarr_create
```
Comment (joshmoore) [inline sources.yml:16]: Why did this get removed, @will-moore ?

**Review by joshmoore — COMMENTED** [review]
Barring one question, I'd say let's get this in and then we can add more and more sophisticated tests.

File: sources.yml (hunk 1 above)
Comment (will-moore) [reply sources.yml:16]: It's duplicated on the next 2 lines. I assumed that was not intended?

Comment (joshmoore) [reply sources.yml:16]: Wow. Ok. Thanks. Maybe I introduced when rebasing? No clue.

**Review by joshmoore — APPROVED** [review]

**Pull request merged**